src/lib/ui/Button.js

```diff
@@ -1,31 +1,136 @@
 const
```

Comment:
Curious about your comments here @gwn. I wanted to create a generic button in order to support most of the buttons in the design. I created three main style tokens container, color, size and in the future I want to generalize this as well because most atomic elements have the same structure but for just used in here instead of theme.

Reply:
Totally supporting this :+1:

Reply:
But the problem is, with your reimplementation of the button component, you seem to have broken the old buttons. You should update the old buttons to comply with your new API.

Reply:
Or maybe you didn't broke them. Not sure at this point.